### PR TITLE
adding test and evaluation field with trainig tfm_ner

### PR DESCRIPTION
**Pull request by SatyamRajawat**
## Describe your changes
Added optional module for testing and evaluation in model training 

## Link to related issue
NA
## Does this introduce new feature or change existing feature of Jaseci? If so, please add tests.
NA
## Will this impact Jaseci users? If so, please write a paragraph describing the impact.
NA
## Anything in this PR should the Jaseci developer/contributor community pay particular attention to? If so, please describe.
NA

### CODE REVIEW

File: jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/entity_extraction.py [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
-import json
-from .train import data_set, check_labels_ok
+from typing import Dict, List, Optional
 import os
 import pandas as pd
-from .entity_utils import create_data, create_data1
+import json
+import warnings
+
+# import sys
```
Comment (AshishMahendra) [inline jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/entity_extraction.py]: remove code that are not required

File: jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/entity_extraction.py [hunk 2]
```diff
@@ -64,38 +137,24 @@ def train(
...
-            print(f"Exception  : {e}")
+    # data = pd.DataFrame(columns=["text", "annotation"])
+    if len(dev_data) != 0:
+        create_train_data(dev_data, "dev")
+
+    if len(test_data) != 0:
+        create_train_data(test_data, "test")
+
+    if len(train_data) != 0:
+        completed = create_train_data(train_data, "train")
```
Comment (AshishMahendra) [inline jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/entity_extraction.py:143]: We'll always need the train file for training, so this should raise exception if null

File: jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/utils/data_tokens.py [hunk 3]
```diff
@@ -47,7 +47,7 @@ def load_data(filename):
 
     labels_name = sorted(list(data.Tag.unique()), reverse=True)
 
-    label2id = {k: v for v, k in enumerate(labels_name)}
-    id2label = {v: k for v, k in enumerate(labels_name)}
+    # label2id = {k: v for v, k in enumerate(labels_name)}
```
Comment (AshishMahendra) [inline jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/utils/data_tokens.py]: please remove unwanted code

**Review by AshishMahendra — CHANGES_REQUESTED** [review]

File: jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/entity_extraction.py (hunk 1 above)
Comment (SatyamRajawat) [reply jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/entity_extraction.py]: done

File: jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/entity_extraction.py (hunk 2 above)
Comment (SatyamRajawat) [reply jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/entity_extraction.py:143]: yes its already raised exception if not get train data

File: jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/utils/data_tokens.py (hunk 3 above)
Comment (SatyamRajawat) [reply jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/utils/data_tokens.py]: done

**Review by AshishMahendra — COMMENTED** [review]
Looks fine to me

**Review by ypkang — APPROVED** [review]

**Pull request merged**